src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -48,21 +54,31 @@ const App = () => (
 );
 
 const MyChild = () => {
-  const { tileId, isVideoEnabled, setIsVideoEnabled, toggleVideo } = useLocalVideo();
+  const { tileId, isVideoEnabled, setIsVideoEnabled, 
+  hasReachedVideoLimit, setHasReachedVideoLimit, toggleVideo } = useLocalVideo();
 
   return (
     <p>Tile ID: {tileId}</p>
 
     <p>
-      {setIsVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+      {isVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+    </p>
+
+    <p>
+      {hasReachedVideoLimit ? 'Video limit reached' : 'Video limit not reached'}
     </p>
 
     <button onClick={() => setIsVideoEnabled(true)}>
       Set isVidelEnabled to true
     </button>
+
+    <button onClick={() => setHasReachedVideoLimit(true)}>
```

Comment:
Wonder why do we want to have example on these setIsVideoEnabled and the new one? I dont think we expect anybody to call them manually right?

Reply:
I am not sure why there is a button to toggle `isVideoEnabled`(may be just want to show the use case of `setIsVideoEnabled`), but I am fine to delete both examples of the `setIsVideoEnabled` button and `setHasReachedVideoLimit` button. What do you think?